src/main/resources/dev.application.properties

```diff
@@ -0,0 +1,4 @@
+jdbc.url=jdbc:postgresql://localhost:5432/agregator-service
+jdbc.user=postgres
+jdbc.password=root
+appPort=8080
```

Comment:
Remove this properties file please
https://devconnected.com/how-to-remove-files-from-git-commit/